actions/upload_sessions.go

```diff
@@ -26,11 +27,11 @@ type uploadSessionCreateReq struct {
 }
 
 type uploadSessionCreateRes struct {
-	ID              string               `json:"id"`
-	UploadSession   models.UploadSession `json:"uploadSession"`
-	BetaSessionID   string               `json:"betaSessionId"`
-	Invoice         models.Invoice       `json:"invoice"`
-	BetaBuriedIndex []int                `json:"betaBuriedIndexes"`
+	ID                string               `json:"id"`
+	UploadSession     models.UploadSession `json:"uploadSession"`
+	BetaSessionID     string               `json:"betaSessionId"`
+	Invoice           models.Invoice       `json:"invoice"`
+	BetaBuriedIndexes []int                `json:"betaBuriedIndexes"`
```

Comment:
Nit: These aren't buried yet, lets rename it to BetaTreasureIndex (same with alpha above)